final/src/main/java/ua/epam/spring/hometask/facade/impl/UserFacadeImpl.java

```diff
@@ -25,4 +27,18 @@
                 .findUserByEmail(email)
                 .map(userConverter::convertToDTO);
     }
+
+    @Override
+    public List<UserDTO> findAll() {
+        return userService.findAll()
+                .stream()
+                .map(userConverter::convertToDTO)
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public UserDTO save(UserDTO user) {
+        User userEntity = userConverter.convertToEntity(user);
+        return userConverter.convertToDTO(userService.save(userEntity));
```

Comment:
I'd recommend to perform saving operations separately. Moreover, saving methods are usually  void.